CHANGELOG.md

```diff
@@ -1,5 +1,68 @@
+ * Loss of completion capability from the client. See[#522](https://github.com/angelozerr/lsp4xml/issues/522)
+ * Error range for `EntityNotDeclared`. See[#518](https://github.com/angelozerr/lsp4xml/issues/518)
+ * Improve revalidate XML files when XSD files is saved. See[#506](https://github.com/angelozerr/lsp4xml/issues/506)
+ * Error range for `src-import.1.2`. See[#499](https://github.com/angelozerr/lsp4xml/issues/499)
+ * Error range for `s4s-elt-invalid-content.3`. See[#496](https://github.com/angelozerr/lsp4xml/issues/496)
+ * NPE in `textDocument/definition` in XSD files. See[#488](https://github.com/angelozerr/lsp4xml/issues/488)
+ * Fix test with markdown on Windows OS.. See[#487](https://github.com/angelozerr/lsp4xml/issues/487)
+ * Error range for `cvc-pattern-valid`. See[#477](https://github.com/angelozerr/lsp4xml/issues/477)
+ * Error range for `AttributePrefixUnbound`. See[#476](https://github.com/angelozerr/lsp4xml/issues/476)
+ * Error range for `EmptyTargetNamespace`. See[#472](https://github.com/angelozerr/lsp4xml/issues/472)
+ * Error range for `ct-props-correct.3`. See[#467](https://github.com/angelozerr/lsp4xml/issues/467)
+ * Error range for `sch-props-correct.2`. See[#462](https://github.com/angelozerr/lsp4xml/issues/462)
+ * Error range for `s4s-elt-must-match.2`. See[#458](https://github.com/angelozerr/lsp4xml/issues/458)
+ * Error range for `ct-props-correct.3`. See[#455](https://github.com/angelozerr/lsp4xml/issues/455)
+ * Error range for `src-ct.1`. See[#453](https://github.com/angelozerr/lsp4xml/issues/453)
+ * Error range for `duplicate attribute`.. See[#452](https://github.com/angelozerr/lsp4xml/issues/452)
+ * Error range for `p-props-correct.2.1`. See[#436](https://github.com/angelozerr/lsp4xml/issues/436)
+ * Element/Attribute names must be case sensitive. See[#433](https://github.com/angelozerr/lsp4xml/issues/433)
+ * Selection formatting ignores attribute indentation preference. See[#429](https://github.com/angelozerr/lsp4xml/issues/429)
+ * Error range for `cos-all-limited.2`. See[#428](https://github.com/angelozerr/lsp4xml/issues/428)
+ * Error range for `src-element.3`. See[#420](https://github.com/angelozerr/lsp4xml/issues/420)
+ * Documents with an Internal Subset DOCTYPE shouldn't stop trying to bind. See[#379](https://github.com/angelozerr/lsp4xml/issues/379)
+ * When DTD is not found, XML is not checked for well-formedness. See[#167](https://github.com/redhat-developer/vscode-xml/issues/167)
+ * unexpected indentation after empty element + attribute. See[#159](https://github.com/redhat-developer/vscode-xml/issues/159)
+ * Formatter inserts spaces in empty lines. See[#157](https://github.com/redhat-developer/vscode-xml/issues/157)
+ * Plugin is not recognized as a XML formatter. See[#154](https://github.com/redhat-developer/vscode-xml/issues/154)
+ * vscode not reloading and revalidating XML/XSD files modified outside itself. See[#131](https://github.com/redhat-developer/vscode-xml/issues/131)
```

Comment:
VSCode was not revalidating XML files when relevant XSD files were modified outside VSCode.